src/components/Inputs/TextInput/TextInput.js

```diff
@@ -54,26 +48,28 @@ function PrivateTextInput({
 
   // Set up validation hooks
   const [getError, setError, validate] = useErrorMessage(validator)
-  const [minMaxValidator] = useMinMaxLength(minLength, maxLength)
```

Comment:
yah, I like how we've consolodated all these into the validator(s) 👍

Reply:
thx yea that was bugging me